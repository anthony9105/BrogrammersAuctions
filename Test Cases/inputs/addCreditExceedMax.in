login nearlyAtMax aaaa
addcredit
10
cancel please
logout