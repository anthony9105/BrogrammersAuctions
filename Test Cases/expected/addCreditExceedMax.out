Login successful
Enter credit to add to user
Error. This transaction will cause credit account to exceed max of 999999999
Transaction cancelled
logout successful
